app/controllers/backend/tax_declarations_controller.rb

```diff
@@ -64,7 +64,12 @@ def index
 
       respond_to do |format|
         format.html do
-          render "alert_no_VAT_declaration" unless FinancialYear.where.not(tax_declaration_mode: "none").any?
+          not_opened_financial_year = FinancialYear.opened.empty?
```

Comment:
Can we rename this into something a bit more english? `no_financial_year_open` maybe?